diff --git a/src/exports/vite/index.js b/src/exports/vite/index.js
index 6d26c63c7c0d2e7a29e27954c7f738fd9204fa2b..89adad6fc2812ccdc06776a5f87d7d9c505f8ba0 100644
--- a/src/exports/vite/index.js
+++ b/src/exports/vite/index.js
@@ -463,7 +463,7 @@ async function kit({ svelte_config }) {
 				// for internal use only. it's published as $app/paths externally
 				// we use this alias so that we won't collide with user aliases
 				case sveltekit_paths: {
-					const { assets, base } = svelte_config.kit.paths;
+					const { assets, base, relative } = svelte_config.kit.paths;

 					// use the values defined in `global`, but fall back to hard-coded values
 					// for the sake of things like Vitest which may import this module
@@ -478,10 +478,10 @@ async function kit({ svelte_config }) {

 					return dedent`
 						export let base = ${s(base)};
-						export let assets = ${assets ? s(assets) : 'base'};
+						export let assets = ${relative ? "'.' + " : ''}${assets ? s(assets) : 'base'};
 						export const app_dir = ${s(kit.appDir)};

-						export const relative = ${svelte_config.kit.paths.relative};
+						export const relative = ${relative};

 						const initial = { base, assets };
